admin/metabox/class-metabox.php

```diff
@@ -140,8 +140,6 @@ public function render_internet_explorer_notice() {
 	 * the main meta box definition array in the class WPSEO_Meta() as well!!!!
 	 */
 	public static function translate_meta_boxes() {
-		WPSEO_Meta::$meta_fields['general']['title']['title'] = __( 'SEO title', 'wordpress-seo' );
```

Comment:
Why did you remove this line?

Reply:
Oops. I think that's a mistake. I'll add it back.